utils/release/git-publish-all.mjs

```diff
@@ -63,33 +97,30 @@ const getCliChangelog = () => {
     await writeChangelog(PATH, changelog);
   }
   updateRootReadme();
-  await gitCommit(
-    dedent`
-    [version bump] chore(release): release ${gitNewTag} [skip ci]
-
-    ${packagesReleased}
-
-    **/README.md
-    **/CHANGELOG.md
-    **/package.json
-    README.md
-    CHANGELOG.md
-    package.json
-    package-lock.json
-    packages/ui/cra-template/template.json
-    `,
-    PATH
+
+  const commit = await commitChanges(
+    releaseNumber,
+    gitNewTag,
+    packagesReleased,
```

Comment:
I think it's not so transparent what these two parameters are for (`gitNewTag` and `packagesReleased`).

Instead, I think we could build the message right here.

For example:
```js
const commit = await commitChanges({
  octokit,
  releaseNumber,
  message: dedent`
    [version bump] chore(release): release ${gitNewTag} [skip ci]
    ${packagesReleased}
    **/README.md
    **/CHANGELOG.md
    **/package.json
    README.md
    CHANGELOG.md
    package.json
    package-lock.json
    packages/ui/cra-template/template.json
  `,
});
```